build-tools/packages/build-cli/src/commands/release/setDtsTypes.ts

```diff
@@ -0,0 +1,182 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { Flags } from "@oclif/core";
+import { Package, updatePackageJsonFile, PackageJson } from "@fluidframework/build-tools";
+import { PackageCommand } from "../../BasePackageCommand";
+import { ExtractorConfig } from "@microsoft/api-extractor";
+import { CommandLogger } from "../../logging";
+import path from "node:path";
+import { strict as assert } from "node:assert";
+import * as fs from "fs";
+
+/**
+ * Represents a list of package categorized into two arrays
+ */
+interface PackageTypesList {
+	/**
+	 * An array of strings containing package names that have been updated.
+	 */
+	packagesUpdated: string[];
+	/**
+	 * An array of strings containing package names that have not been updated.
+	 */
+	packagesNotUpdated: string[];
+}
+
+const knownDtsKinds = ["alpha", "beta", "public", "untrimmed"] as const;
+
+type DtsKind = typeof knownDtsKinds[number];
+
+function isDtsKind(str: string | undefined): str is DtsKind {
+	return str === undefined ? false : knownDtsKinds.includes(str as any);
+}
+
+export default class SetReleaseTagPublishingCommand extends PackageCommand<
+	typeof SetReleaseTagPublishingCommand
+> {
+	static readonly description =
+		"Updates the .d.ts types that are included in the `types` field in package.json. This command is intended to be used in CI only.";
```

Comment:
THis makes it sound like it modifies the d.ts files. I think this would be more clear:

```suggestion
		"Updates which .d.ts file is referenced by the `types` field in package.json. This command is used during package publishing (by CI) to select the d.ts file which corresponds to the selected API-Extractor release tag.";
```